### PR TITLE
Refactoring of admin docs

### PR DESCRIPTION
**Pull request by pameladelgado**
Goes in tandem with https://github.com/SwissDataScienceCenter/renku/pull/633
Still need to reorganize the openstack directory

### CODE REVIEW

File: renku-values.yaml [hunk 1]
```diff
@@ -284,9 +300,37 @@ notebooks:
...
         cryptoKey: *nb_jupyterhub_auth_state_cryptoKey
       gitlab:
+        clientId: *nb_jupyterhub_auth_gitlab_clientId
         clientSecret: *nb_jupyterhub_auth_gitlab_clientSecret
+        callbackUrl: *jupyterhub_gitlab_callbackUrl
 
+graph:
+  enabled: true
+  jena:
+    admin:
```
Comment (rokroskar) [inline renku-values.yaml]: this is not current

File: renku-values.yaml [hunk 2]
```diff
@@ -284,9 +300,37 @@ notebooks:
...
+      password: *sparql_password
+    persistence:
+      existingClaim: renku-jena ## Alternatively, replace with the PVC name
+  gitlab:
+    url: *gitlabURL
+  webhookService:
+    hookToken:
+      secret: YzBjODA4NTlhZjUwODkxOQo=  ## Genereated with openssl rand -hex 8|base64
```
Comment (rokroskar) [inline renku-values.yaml]: don't pre-fill these

File: renku-values.yaml [hunk 3]
```diff
@@ -1,38 +1,45 @@
...
-  pass401: &postgresql_postgresPassword                     ## ReplaceMeWithA64CharString
-  pass501: &nb_jupyterhub_hub_cookieSecret                  ## ReplaceMeWithA64CharString
-  pass502: &nb_jupyterhub_hub_services_notebooks_apiToken   ## ReplaceMeWithA64CharString
-  pass503: &nb_jupyterhub_hub_services_gateway_apiToken     ## ReplaceMeWithA64CharString
-  pass504: &nb_jupyterhub_proxy_secretToken                 ## ReplaceMeWithA64CharString
-  pass505: &nb_jupyterhub_auth_state_cryptoKey              ## ReplaceMeWithA64CharString
-  pass506: &nb_jupyterhub_auth_gitlab_clientSecret          ## ReplaceMeWithA64CharString
-  pass601: &gateway_secretKey                               ## ReplaceMeWithA64CharString
-  # pass701: &graph_token_repo                                ## ReplaceMeWithA64CharString
+  pass101: &global_gitlab_clientSecret                      ## Replace with gitlab client in keycloak
```
Comment (rokroskar) [inline renku-values.yaml]: you don't need to "replace" here, you need to add the secrets/tokens, no?

File: renku-values.yaml [hunk 4]
```diff
@@ -213,8 +226,11 @@ ui:
       We also kindly request that you do not upload datasets larger than 1GB for
       the time being. In the future, we will remove this restriction.
 
-      ### Renku version 0.3.1 (March 11th 2019)
-      
+      ### Renku version 0.4.0 (May 28th 2019)
```
Comment (rokroskar) [inline renku-values.yaml]: We can leave this text more generic

File: renku-version.txt [hunk 5]
```diff
@@ -1 +1 @@
-0.3.1
+0.4.0
```
Comment (rokroskar) [inline renku-version.txt]: not current

**Comment by pameladelgado** [comment]
Thanks @rokroskar I applied the suggestions.
Next in the pipeline: 
* create a script to generate a default, minimal renku-values.yaml file.
* clean up renku-values.yaml file after helm charts refactoring.

**Pull request merged**